### PR TITLE
added File enrichment

### PR DESCRIPTION
**Pull request by k1r10n**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: only docker image updated

## Description
Add a new command + update docs

## Screenshots
xsoar-rst-pack-1 1 0

## Minimum version of Cortex XSOAR
- [ X ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ X ] Yes
       - Further details:
   - [ ] No

## Must have
- [ X ] Tests
- [ X ] Documentation

### CODE REVIEW

**Comment by k1r10n** [comment]
Hi @MLainer1, please advise me if anything else is needed to be updated

**Review by MLainer1 — CHANGES_REQUESTED** [review]
Good work!
I'm not sure if the data from the test data is real or not, make sure to mock it so you won't have secrets in the pack.

File: Packs/RSTCloud/Integrations/RSTCloudThreatFeedAPI/RSTCloudThreatFeedAPI.py [hunk 1]
```diff
@@ -1,689 +1,859 @@
...
+            human_readable_score = ''
+            if calc_score == 3:
+                human_readable_score = 'Malicious'
+            if calc_score == 2:
+                human_readable_score = 'Suspicious'
+
+            table = {'Score': total_score,
+                     'Relevance': human_readable_score,
+                     'Threat': ', '.join(threat for threat in indicator.get('threat', '')),
+                     'Last Seen': time.strftime('%Y-%m-%d', time.localtime(int(indicator.get('lseen', '')))),
```
[the remark this answers is not in the document]
Comment (k1r10n) [reply Packs/RSTCloud/Integrations/RSTCloudThreatFeedAPI/RSTCloudThreatFeedAPI.py]: fixed int() and mocked the data

**Comment by k1r10n** [comment]
thanks @MLainer1. Anything else to be updated from our side?

**Comment by MLainer1** [comment]
> thanks @MLainer1. Anything else to be updated from our side?

No, let's schedule a demo, I'll send you a message

**Comment by tomer-pan** [comment]
Hi @k1r10n 
Per our last demo, I remember we agreed on 2 changes:
1) Use the `IsIntegrationAvailable` automation for checking if the instance is active.
2) Change the "File Enrichment" playbook input to accept hashes and not a file object.

I also told you that I will check whether it's possible to check all of the inputs with one task and seems like you can use the `SetIfEmprty` transformer in order to choose the first supplied value.
So for example if you have sha256/sha1/md5 as playbook inputs, you would first check if either of them exist:
image

And then you would run the "file" command with either one of them which exist:
image

**Comment by k1r10n** [comment]
Thank you for the update. Incorporated `IsIntegrationAvailable` function and `SetIfEmpty` as suggested. However, I kept File as input and then checking for md5/sha25/sha1 existence 


image
image

image

**Comment by k1r10n** [comment]
As it is designed to be working as a subplaybook, it should be easier to invoke it and pass File to the input rather than creating a complex input definition to pass all possible hash types: md5, sha1, sha256

**Comment by tomer-pan** [comment]
> As it is designed to be working as a subplaybook, it should be easier to invoke it and pass File to the input rather than creating a complex input definition to pass all possible hash types: md5, sha1, sha256

Oh okay, I just saw that you are referring to the hashes through the "File" object. Im not sure it's the best idea since it is limiting (if no File object exist, it will not run although you might have hashes in other keys/objects in the context in some scenarios) but this is your call.

**Comment by k1r10n** [comment]
yes, please see the first step, it checks File object existence as well

**Review by tomer-pan — APPROVED** [review]

**Comment by k1r10n** [comment]
@MLainer1 Yes, please, merge it. The other scenario is covered by the default "File Enrichment - File reputation" playbook which accepts hashes as an input and uses the 'FileReputation' automation script that covers the case mentioned by @tomer-pan 

Thank you for your support!

**Review by MLainer1 — APPROVED** [review]

**Pull request merged**